src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/__test__/RoleForm.test.js

```diff
@@ -82,15 +81,44 @@ describe('/signals/settings/roles/components/RoleForm', () => {
     });
   });
 
+  it('should handle submit flow when saving an new role', () => {
+    props.role = undefined;
```

Comment:
You're changing the `prop` var here. Better to use
```jsx
const propsWithoutRole = { ...props, role: undefined };
const { getByTestId } = render(withAppContext(<RoleForm {...propsWithoutRole} />));
```

Reply:
dat hoeft niet want in de beforeEach wordt hij elke keer de props gezet

Reply:
gedaan